Python/pywarpx/picmi.py

```diff
@@ -1808,8 +1804,7 @@ def initialize_inputs(self):
         self.diagnostic.plot_raw_fields_guards = self.plot_raw_fields_guards
         self.diagnostic.plot_finepatch = self.plot_finepatch
         self.diagnostic.plot_crsepatch = self.plot_crsepatch
-        self.diagnostic.write_species = self.write_species
-
+        self.diagnostic.write_species = False
```

Comment:
Thanks for this PR. Note that care is needed here since `FieldDiagnostic` and `ParticleDiagnostic` can be used to setup the same diagnostic, e.g. `diag1`. The `ParticleDiagnostic` should set `write_species` to True, and this should only set it to false if it hadn't already been set. Something like this
```
if 'write_species' not in self.diagnostic.argvattrs:
    self.diagnostic.write_species = False
```